mappings/net/minecraft/util/collection/Weighted.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_6008 net/minecraft/util/collection/Weighted
+	METHOD method_34979 getWeight ()Lnet/minecraft/class_6007;
+	METHOD method_34980 of (Ljava/lang/Object;I)Lnet/minecraft/class_6008$class_6010;
+		ARG 0 data
+		ARG 1 weight
+	CLASS class_6009 Absent
+		FIELD field_29931 weighting Lnet/minecraft/class_6007;
```

Comment:
Should this be `weight` now that `Weighting` is `Weight`?

Reply:
```suggestion
		FIELD field_29931 weight Lnet/minecraft/class_6007;
```